airflow-core/src/airflow/api_fastapi/core_api/routes/public/task_instances.py

```diff
@@ -770,6 +770,9 @@ def post_clear_task_instances(
         )
 
     if not dry_run:
+        if hasattr(body, 'is_running_message'):
```

Comment:
The `body` is already a Pydantic schema. Would it be better to access the attribute directly?
```suggestion
        if body.is_running_message:
```